typhon/arts/griddedfield.py

```diff
@@ -82,7 +82,7 @@ def __getitem__(self, index):
 
     def __eq__(self, other):
         """Test the equality of GriddedFields."""
-        if isinstance(other, self.__class__):
+        if isinstance(other, self.__class__) or isinstance(self, other.__class__):
```

Comment:
To improve clarity (and keep the width below 80 characters :wink:), this can be written as:

```python
if isinstance(other, (self.__class__, other.__class__)):
    ...
```